game/osu/String.ksy

```diff
@@ -0,0 +1,16 @@
+meta:
+  id: osu_string
+  endian: le
+  imports:
+    - /common/vlq_base128_le
+seq:
+  - id: indicator
+    type: u1
+  - id: string_length
+    type: vlq_base128_le
+    if: indicator == 0x0b
+  - id: value
+    type: str
+    encoding: UTF-8

```

Comment:
can be put into meta

Reply:
what you mean by putting into meta?

Reply:
you can set the default encoding for strings in a type by setting it in `meta`